plugins/suggested-blocks/src/index.js

```diff
@@ -0,0 +1,204 @@
+/**
+ * @license
+ * Copyright 2012 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Utility functions for handling suggestions.
+ */
+'use strict';
+
+/**
+ * Utility functions for handling block suggestions.
+ * @namespace Blockly.SuggestedBlocks
+ */
+import * as Blockly from 'blockly';
+
+const NUM_BLOCKS_PER_CATEGORY = 10;
```

Comment:
How would you feel about making this configurable? E.g. via a `config` parameter to the `init` method?